widget/edit.controller.js

```diff
@@ -2,23 +2,25 @@
 (function () {
     angular
         .module('cybersponse')
-        .controller('editCustomPicklistMessage110Ctrl', editCustomPicklistMessage110Ctrl);
+        .controller('editCustomPicklistMessageDev200Ctrl', editCustomPicklistMessageDev200Ctrl);
+        
+        editCustomPicklistMessageDev200Ctrl.$inject = ['$scope', '$uibModalInstance', '$state', 'config', 'Entity', 'picklistsService', '_'];
 
-        editCustomPicklistMessage110Ctrl.$inject = ['$scope', '$uibModalInstance', '$state', 'config', 'Entity', 'picklistsService', '_'];
-
-    function editCustomPicklistMessage110Ctrl($scope, $uibModalInstance, $state, config, Entity, picklistsService, _) {
+    function editCustomPicklistMessageDev200Ctrl($scope, $uibModalInstance, $state, config, Entity, picklistsService, _) {
         $scope.addOptions = addOptions;
         $scope.removeOptions = removeOptions;
         $scope.cancel = cancel;
         $scope.save = save;
         $scope.config = config;
+        $scope.config.isTimeout = config.isTimeout || false
         $scope.config.picklits_mapping = $scope.config.picklits_mapping || {
         options: []
        };
-      
+       
         function _init() {
             if ($state.params.module) {
                 loadAttributes();
+                loadAttributesToUpdate();
```

Comment:
Do we really need this method? As same can be processed as a part of loadAttributes() method. Here code redundancy can be avoided.